TodoApp.ConsoleApp/Repositories/XMLRepository/TodoRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Xml.Linq;
+using TodoApp.ConsoleApp.Repositories.Models;
+using TodoApp.ConsoleApp.Repositories.Interfaces;
+
+namespace TodoApp.ConsoleApp.Repositories.XMLRepository
+{
+    internal class TodoRepository : RepositoryBase<TodoModel, Guid>, ITodoRepository
```

Comment:
Make all classes and interfaces in the Repository module public.
They are intended to be used to access data from everywhere. We should not limit they usage.